lib/std/builtin.zig

```diff
@@ -174,6 +174,10 @@ pub const AddressSpace = enum {
     param,
     shared,
     local,
+
+    // AVR address spaces.
+    data,
```

Comment:
Considering that data is the default, should that be exposed? its the its the same as setting it to `.generic`

Reply:
I think it'd probably be better to have fewer duplicates. Also is there a reason we only expose one flash address space unlike clang and gcc?

Reply:
Last time i checked (~1y ago) it only supported the one

Reply:
https://github.com/llvm/llvm-project/blob/44fe4e25e45c1999f9d45e20724beb38ae505dfb/llvm/lib/Target/AVR/AVR.h#L41 ah, seems that theres more support now. Unfortunately it doesnt add the linearalized address space yet